.ci/kitchen-centos7-py2

```diff
@@ -46,6 +46,8 @@ pipeline {
                         }
                     }}
                     archiveArtifacts artifacts: 'artifacts/xml-unittests-output/*.xml'
+                    archiveArtifacts artifacts: 'artifacts/logs/minion'
+                    archiveArtifacts artifacts: 'artifacts/coverage/coverage.xml
```

Comment:
coverage.xml should be pushed to the coverage plugin, we need `salt-runtests.log.